### ISSUE TITLE
Game list... launch or edit?

### ISSUE THREAD

**Issue by Flieh** [zond/android-diplicity#41]
A few players have brought this up. Would  it not be simpler (player perspective of course, not dev) if the game list functioned thus:
- **simple press** opens the game completely. 
- **long press** opens up the expanded view for roster and alias editing.

**Comment by zond** [comment]
I hate long press - it's even more undiscoverable than swiping :/

**Comment by Flieh** [comment]
yeah, the long press itself isn't really the issue I guess. It's that in order to open your game you have to press it once and then press the **four arrow** icon. That seems like too much.

**Comment by zond** [comment]
Still faster than a long press..

**Comment by GijsTimmers** [comment]
I would say:

Short press: open the game completely
Three small dots button on the right side: open settings
![vevdm](https://user-images.githubusercontent.com/972314/29585321-0321fd2c-8787-11e7-8b66-6f75baad3e75.png)

**Comment by zond** [comment]
Sure, that would work I guess.

Anyone willing to take a grip on the UI/UX has a good chance to just design
it completely the way they want :)

**Comment by flurbius** [comment]
I would love to take you up on that, Im frustrated by my lack of decent dev pc, someday I will get passed this road block and surprise you all.

I think long press is a first class UI channel that we should use, it is useful and easy to do and remember and saves screen real estate.

I know some people dont like it because its obscure - all that needs to happen is for the user to be told about it, after finding anything at all useful by long clicking once, the user will become enlightened and start long pressing everything in the hope of solving their needs.  

We could put up some simple UI instructions on the first run or later by demand from a menu item or when the user tries something that doesnt work.
